examples/source_stream_wfs_25d.html

```diff
@@ -219,17 +217,15 @@
                 },
             });
 
-            var wfsBuildingLayer = new itowns.GeometryLayer('wfsBuilding', new itowns.THREE.Group(), {
+            var wfsBuildingLayer = new itowns.FeatureGeometryLayer('wfsBuilding', {
                 update: itowns.FeatureProcessing.update,
```

Comment:
Could you also remove the `update` parameter ?

Reply:
yes